Easydict/Feature/ViewController/View/Titlebar/EZTitlebar.m

```diff
@@ -145,7 +145,7 @@ - (void)updateConstraints {
         EZOpenLinkButton *eudicButton = [[EZOpenLinkButton alloc] init];
         
         // !!!: Note that some applications have multiple channel versions. Ref: https://github.com/tisfeng/Raycast-Easydict/issues/16
-        BOOL installedEudic = [self checkInstalledApp:@[@"com.eusoft.freeeudic", @"com.eusoft.eudic"]];
+        BOOL installedEudic = [self checkInstalledApp:@[@"com.eusoft.freeeudic", @"com.eusoft.eudic", @"eusoft.eudic.ip"]];
```

Comment:
Thank you very much for the PR. 

I would like to ask which version of Eudic is the Bundle ID of `eusoft.eudic.ip`?

Reply:
The iPad version contains widgets and can run on Apple silicon, that is why I use it replaced from the macOS version.


https://apps.apple.com/cn/app/%E6%AC%A7%E8%B7%AF%E8%8B%B1%E8%AF%AD%E8%AF%8D%E5%85%B8-eudic-%E6%B1%89%E8%8B%B1%E8%8B%B1%E6%B1%89%E4%BA%92%E8%AF%91%E5%B7%A5%E5%85%B7/id367278030

Reply:
Ok, thanks.